packages/core/README.md

```diff
@@ -132,6 +134,21 @@ switchTheme('custom', document.querySelector('.section'));
 
 If you do not use JavaScript, you might consider including a different CSS theme file based on URL parameter.
 
+Lastly, if you're extremely concerned about efficiency, you can shave off 1-3 KBs by not including base tokens twice, if they're shared between the themes you're switching.
+
+```scss
+@use '~@onfido/castor';
+
+:root {
+  @include castor.tokens();
+}
+
+:global {
```

Comment:
Here yet again using `:global` where as other examples are not using CSS modules.

Reply:
Removed.